river/tree/_nodes/efdtc_nodes.py

```diff
@@ -123,44 +114,31 @@ class EFDTSplitNode(SplitNode, BaseEFDTNode):
         Class observations
     depth
         The depth of the node in the tree.
-    attr_obs
+    splitter
         The numeric attribute observer algorithm used to monitor target statistics
         and perform split attempts.
-    attr_obs_params
-        The parameters passed to the numeric attribute observer algorithm.
-    attribute_observers
+    existing_splitters
         Existing attribute observers from previous nodes passed to provide a warm-start.
+    **kwargs
+        Other parameters passed to the learning nodes.
```

Comment:
What is the correct choice for docstrings? `kwargs` or  `**kwargs`?

Reply:
`kwargs`

Reply:
Thanks! I corrected the docstrings in commit [d6931d9](https://github.com/online-ml/river/pull/488/commits/d6931d9e1632978aeb7276b9a13cff1ac2a3ec09).